.github/workflows/production.yml

```diff
@@ -110,7 +110,6 @@ jobs:
       matrix:
         config:
           - { os: ubuntu-latest, target: x86_64-unknown-linux-gnu, cross: false }
-          - { os: ubuntu-latest, target: aarch64-unknown-linux-gnu, cross: true }
```

Comment:
do we need to depreciate support for aarch64?

Reply:
I just realized that I made a mistake, I will implement the correct build in the next PR.